raiden/encoding/messages.py

```diff
@@ -222,6 +227,23 @@ def cmdid(id_):
 )
 
 
+RequestMonitoring = namedbuffer(
+    'request_monitoring_service',
```

Comment:
Can you change the name to `request_monitoring` here? All other messages have the same name for the named buffer object (in Camelcase) and for the namedbuffer string.